addons/api/addon/models/worker.js

```diff
@@ -55,6 +65,34 @@ export default class WorkerModel extends GeneratedWorkerModel {
     );
   }
 
+  /**
+   * Returns the api tags as an array of key/value pair objects.
+   * @type {[object]}
+   */
+  getApiTagList() {
+    if (!this.api_tags) {
+      return null;
+    }
+
+    return Object.entries(this.api_tags).flatMap(([key, value]) =>
+      value.map((tag) => ({ key, value: tag })),
+    );
+  }
+
+  /**
+   * Returns all tags as an array of key/value pair objects with tag type.
+   * @type {[object]}
+   */
+  getAllTags() {
+    return [
+      ...(this.getConfigTagList() || []).map((tag) => ({
+        ...tag,
+        type: 'config',
+      })),
+      ...(this.getApiTagList() || []).map((tag) => ({ ...tag, type: 'api' })),
+    ];
```

Comment:
Is this needed? Can we just look at `canonical_tags` instead? That should be a union of unique config tags and api tags

Reply:
Just relying on `canonical_tags` doesn't give us insight into what kind of tag it is. Only api tags can be created or removed and we need the type to determine what actions the user can take for each tag. In the below API response, canonical_tags doesn't give me all the info I need.
```json
{
    "canonical_tags": {
        "boundary.cloud.hashicorp.com:managed": [
            "true"
        ],
        "key1": [
            "dev"
        ],
        "key2": [
            "prod",
            "test"
        ]
    },
    "config_tags": {
        "boundary.cloud.hashicorp.com:managed": [
            "true"
        ]
    },
    "api_tags": {
        "key1": [
            "dev"
        ],
        "key2": [
            "prod",
            "test"
        ]
    },
}
```

Reply:
I see, makes sense then.